examples/exp_configs/rl/multiagent/multiagent_i210.py

```diff
@@ -118,7 +125,7 @@
     # simulation-related parameters
     sim=SumoParams(
         sim_step=0.1,
-        render=False,
+        render=True,
```

Comment:
I think this should probably be false by default and accept a command line arg.

Reply:
can't add command line flags in this file, it's supposed to be default params. run script can make it false.